debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbConnectorConfig.java

```diff
@@ -236,47 +239,89 @@ public static SnapshotMode parse(String value, String defaultValue) {
             .withImportance(Importance.HIGH)
             .withValidation(Field::isListOfRegex)
             .withDescription("The databases for which changes are to be excluded");
 
+    /**
+     * Old, backwards-compatible "blacklist" property.
+     */
+    public static final Field DATABASE_BLACKLIST = Field.create("database.blacklist")
+            .withDisplayName("DB Blacklist")
+            .withType(Type.LIST)
+            .withWidth(Width.LONG)
+            .withImportance(Importance.LOW)
+            .withValidation(Field::isListOfRegex)
+            .withDescription("The databases for which changes are to be excluded (deprecated, use \"" + DATABASE_EXCLUDE_LIST.name() + "\" instead)");
+
     /**
      * A comma-separated list of regular expressions that match the fully-qualified namespaces of collections to be monitored.
      * Fully-qualified namespaces for collections are of the form {@code <databaseName>.<collectionName>}.
-     * May not be used with {@link #COLLECTION_BLACKLIST}.
+     * Must not be used with {@link #COLLECTION_EXCLUDE_LIST}.
      */
-    public static final Field COLLECTION_WHITELIST = Field.create("collection.whitelist")
-            .withDisplayName("Collections")
+    public static final Field COLLECTION_INCLUDE_LIST = Field.create("collection.include.list")
+            .withDisplayName("Collections Include List")
             .withType(Type.LIST)
             .withWidth(Width.LONG)
             .withImportance(Importance.HIGH)
             .withValidation(Field::isListOfRegex,
-                    MongoDbConnectorConfig::validateCollectionBlacklist)
+                    MongoDbConnectorConfig::validateCollectionExcludeList)
             .withDescription("The collections for which changes are to be captured");
 
+    /**
+     * Old, backwards-compatible "whitelist" property.
+     */
+    public static final Field COLLECTION_WHITELIST = Field.create("collection.whitelist")
+            .withDisplayName("Collections Whitelist")
+            .withType(Type.LIST)
+            .withWidth(Width.LONG)
+            .withImportance(Importance.LOW)
+            .withValidation(Field::isListOfRegex,
+                    MongoDbConnectorConfig::validateCollectionExcludeList)
+            .withDescription("The collections for which changes are to be captured (deprecated, use \"" + COLLECTION_INCLUDE_LIST.name() + "\" instead)");
+
     /**
      * A comma-separated list of regular expressions that match the fully-qualified namespaces of collections to be excluded from
      * monitoring. Fully-qualified namespaces for collections are of the form {@code <databaseName>.<collectionName>}.
-     * May not be used with {@link #COLLECTION_WHITELIST}.
+     * Must not be used with {@link #COLLECTION_INCLUDE_LIST}.
+     */
+    public static final Field COLLECTION_EXCLUDE_LIST = Field.create("collection.exclude.list")
+            .withValidation(Field::isListOfRegex)
+            .withInvisibleRecommender();
```

Comment:
@rk3rn3r Should not be invisible blacklist, not exclude?

Reply:
both are invisible! see the next few lines

Reply:
I really wonder what was the rationale behind having blacklist invisible. But let's keep it as is now.

Reply:
The reason was that it is not shown/recommended because it can be invalid to set a blacklist when a whitelist is already defined (at least according to the docs of InvisivbleRecommender). I think there should be a DependentRecommender that automatically shows or hides based on predicates that work with the provided config?